### DISCUSSION TITLE
How to add your own objects.

### DISCUSSION THREAD

**Discussion by grzesczwa** [Stellarium/stellarium#3631] [Q&A]
How to add your own objects, e.g. stars from catalog UCAC4, to the program.

I thought that the new Missing Stars module would solve my problem.
Unfortunately, after installing the new Stellarium version 23.4  and running the program, the program does not create the MissingStars directory in C:\Users\<users>\AppData\Roaming\Stellarium\modules and does not add the missingstars.json file. Even if I add a directory and file manually. These objects are not visible.

**Comment by gzotti** [comment]
UCAC4 is currently not an option. The MissingStars plugin adds a few handfuls of extra stars that have been reported as missing, but can currently not be extended by the user.

**Comment by grzesczwa** [comment]
Thank you for your answer


I wanted to write here why I need the program to have the functionality of adding other stars.

In 2020 I wrote the Cartes du Ciel (Sky Charts) Addin for Occult Watcher. The add-in allows you to pass data from Occult Watcher to display data in Sky Chart.

The description of the Cartes du Ciel (Sky Charts) Addin for Occult Watcher is available at https://www.sopiz.ptma.pl/cartes-du-ciel-sky-charts-addin-for-occult-watcher.

You can read more about the OccultWatcher program on the program website https://www.occultwatcher.net.

A few months ago I wrote another Stellarium Addin for Occult Watcher. Unfortunately, I am not completely happy that the program supports the ability to add your own stars.

Best wishes
Grzegorz Czepiczek

**Reply by gzotti** [reply]
If you just need to see more stars, you can activate the DSS image layer. If you need more clickable stars with information, somebody must convert UCAC4 into our star catalog format. Probably "just" replace NOMAD by UCAC4, and discard stars already included in the brighter catalog files.
